### PR TITLE
Add markdown links check

### PR DESCRIPTION
**Pull request by d-uzlov**
Closes https://github.com/mlcommons/mobile_app_open/issues/209

### CODE REVIEW

**Review by anhappdev — APPROVED** [review]

File: tools/formatter/format.mk [hunk 1]
```diff
@@ -72,6 +72,10 @@ lint/line-endings:
 	else echo all files have unix line endings; \
 	fi
 
+.PHONY: lint/markdown-links
+lint/markdown-links:
```
Comment (anhappdev) [inline tools/formatter/format.mk:76]: Just a note. This one `lint/markdown-links` will not be called if using `docker/format`.

Comment (d-uzlov) [reply tools/formatter/format.mk:76]: I know. Same as with all other `lint` make targets (and some of then are not covered by the `format` target, just like this one).
I have few ideas on how to improve it, but I'm yet to formalize them to create an issue for it.

Comment (anhappdev) [reply tools/formatter/format.mk:76]: Ok. Because I see you added the tool to `Dockerfile` without using it, so just a note.

Comment (d-uzlov) [reply tools/formatter/format.mk:76]: Yeah, I launched the docker manually for this

**Pull request merged**